src/components/audio-player/AudioPlayer.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react'
+import { ActivityIndicator, Image, SafeAreaView, Text, View } from 'react-native'
+import TrackPlayer, { Track, useActiveTrack } from 'react-native-track-player'
+
+import {
+  DownloadButton,
+  FullScreenButton,
+  LoopButton,
+  PlayPauseButton,
+  Progress,
+  SkipButton,
+  SpeedMenu,
+  VolumeControl,
+} from '@components'
+import { PLAYBACK_TRACKS } from '@constants'
+
+import { styles } from './audioPlayer-styles'
+
+const AudioPlayer = () => {
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+  const [currentTrack, setCurrentTrack] = useState<Track>()
+
+  async function setPlayer() {
```

Comment:
somewhere we are using arrow functions and somewhere we are using function keyword, use only one, can prefer arrow function everywhere